XCUITests/WebAuthenticationTest.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WebAuthenticationTest.swift
+//  Client
+//
+//  Created by mozilla on 12/23/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
+import XCTest
+import GCDWebServers
+
+class WebAuthenticationTest: BaseTestCase {
+private var webRoot: String!
+        
+    override func setUp() {
+       
+        super.setUp()
+        dismissFirstRunUI()
+        webRoot = SimplePageServer.start()
+        continueAfterFailure = false
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+
+    private func loadAuthPage() {
+        let app = XCUIApplication()
+
+        app.textFields["url"].tap()
+        app.textFields["address"].typeText("\(webRoot)/auth.html\n")
```

Comment:
@farhanpatel I copied the SimplePageServer class from Global.swift to BaseTestCase.swift, and followed the GCDWebServer's instruction to copy the Carthage framework, but the server does not seem to be created. (the webServer.port is always set to 0, and I don't see any error messages) I was wondering whether you can help me out figuring what I have missed?